### PR TITLE
Spacy and stealth browsers

### LINKED ISSUE #2889: spacy dependencies failure
- Nuitka version, full Python version, flavor, OS, etc. as output by *this exact* command.

2.2.3
Commercial: 2.5.1
Python: 3.11.9 (main, Apr 19 2024, 16:39:34) [GCC 11.2.0]
Flavor: Anaconda Python
Executable: /home/orangepi/miniconda3/envs/theplatform_ov_prod/bin/python
OS: Linux
Arch: aarch64
Distribution: Ubuntu (based on Debian) 22.04.3
Version C compiler: /usr/bin/gcc (gcc 11).

- How did you install Nuitka and Python

python -m pip install -U --force-reinstall https://XXX@github.com/Nuitka/Nuitka-commercial/archive/main.zip

- The specific PyPI names and versions

...
blis                      0.7.11
Nuitka                    2.2.3
preshed                   3.0.9
spacy                     3.7.4
spacy-legacy              3.0.12
spacy-loggers             1.0.5
thinc                     8.2.3
...

- Also supply a Short, Self Contained, Correct, Example

Simple program `spacy_test.py`:
```
import spacy

nlp = spacy.load("en_core_web_sm", disable=["ner", "parser"])
tokens = list(nlp("This is a very good test."))
print([(token.text, token.pos_) for token in tokens])
```

Expected output:
`[('This', 'DET'), ('is', 'AUX'), ('a', 'DET'), ('very', 'ADV'), ('good', 'ADJ'), ('test', 'NOUN'), ('.', 'PUNCT')]`

Compile command (I came up with this based on other issues filed related to spacy):
`python -m nuitka --follow-imports --standalone --include-package=spacy,en_core_web_sm,thinc,blis,preshed  --include-package-data=spacy,en_core_web_sm  spacy_test.py`

Even with the command above, there were lots of missing dependencies, and I had to do the following:
```
cd spacy_test.dist/
SITE_PKGS_DIR=/home/orangepi/miniconda3/envs/myconda/lib/python3.11/site-packages

cp $SITE_PKGS_DIR/thinc/backends/_custom_kernels.cu thinc/backends/
cp $SITE_PKGS_DIR/thinc/backends/linalg.cpython-311-aarch64-linux-gnu.so thinc/backends/
cp $SITE_PKGS_DIR/spacy/parts_of_speech.cpython-311-aarch64-linux-gnu.so spacy/
cp $SITE_PKGS_DIR/spacy/default_config.cfg spacy/
cp $SITE_PKGS_DIR/spacy/pipeline/transition_parser.cpython-311-aarch64-linux-gnu.so spacy/pipeline/
cp $SITE_PKGS_DIR/spacy/pipeline/_parser_internals/stateclass.cpython-311-aarch64-linux-gnu.so spacy/pipeline/_parser_internals/
cp $SITE_PKGS_DIR/spacy/pipeline/_parser_internals/transition_system.cpython-311-aarch64-linux-gnu.so spacy/pipeline/_parser_internals/
cp $SITE_PKGS_DIR/spacy/pipeline/_parser_internals/_beam_utils.cpython-311-aarch64-linux-gnu.so spacy/pipeline/_parser_internals/
cp $SITE_PKGS_DIR/spacy/pipeline/_parser_internals/arc_eager.cpython-311-aarch64-linux-gnu.so spacy/pipeline/_parser_internals/
cp $SITE_PKGS_DIR/spacy/pipeline/_parser_internals/ner.cpython-311-aarch64-linux-gnu.so spacy/pipeline/_parser_internals/
mkdir -p spacy/cli/templates/
cp $SITE_PKGS_DIR/spacy/cli/templates/quickstart_training_recommendations.yml spacy/cli/templates/
mv en_core_web_sm/en_core_web_sm-3.7.1/* en_core_web_sm/
```

Now, I am still stuck, with this error:
```
Traceback (most recent call last):
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy_test.py", line 3, in <module>
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy/__init__.py", line 51, in load
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy/util.py", line 467, in load_model
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy/util.py", line 547, in load_model_from_path
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy/language.py", line 2209, in from_disk
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy/util.py", line 1390, in from_disk
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy/language.py", line 2203, in <lambda>
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy/pipeline/lemmatizer.py", line 304, in from_disk
  File "/home/orangepi/platform/theplatform/spacy_test.dist/spacy/pipeline/lemmatizer.py", line 173, in _validate_tables
ValueError: [E912] Failed to initialize lemmatizer. Missing lemmatizer table(s) found for mode 'rule'. Required tables: ['lemma_rules']. Found: [].
```

- Consider getting commercial support

Yes, I am a commercial customer.

### PR DESCRIPTION
**Pull request by KRRT7**
fixes #2889 
adds new plugin for spacy and kwarg for the plugin as `--spacy-language-model=model_name`

### CODE REVIEW

File: nuitka/plugins/standard/standard.nuitka-package.config.yml [hunk 1]
```diff
@@ -1200,17 +1218,17 @@
...
     - description: 'use SGML code more naturally'
       context:
         - 'import inspect'
         - 'import textwrap'
         - 'import sgmllib'
-      replacements_plain:
-        '__parse_starttag.__code__ = sgmllib.SGMLParser.parse_starttag.__code__': 'pass'
-        'def goahead(self, i):': 'def goahead(self, end):'
-        'goahead.__code__ = sgmllib.SGMLParser.goahead.__code__': 'pass'
+      replacements:
```
Comment (kayhayen) [inline nuitka/plugins/standard/standard.nuitka-package.config.yml:1228]: Where is that change coming from, very strange, merge problem?

**Review by kayhayen — CHANGES_REQUESTED** [review]
Seems I need to manually fix this one for the Yaml changes, can it be that you just copied a stable file over to develop, or a merge issue of some sorts?

**Comment by KRRT7** [comment]
the changes came from running the yaml formatter, before that i wasn't seeing any merge conflicts and it was a fresh branch of develop

**Comment by kayhayen** [comment]
Strange stuff, but no big deal, I will be able to push to it later today and then merge for the next hotfix.

**Comment by kayhayen** [comment]
As agreed, I am making this a manual merge, and updating the plugin code with distribution scans and checks for models, so we can handle them automatically and error out if the user has no language models selected.

**Pull request closed without merging**